e2e/v8-coverage/no-sourcemap/package.json

```diff
@@ -4,8 +4,7 @@
   "jest": {
     "testEnvironment": "node",
     "transform": {
-      "\\.[jt]sx?$": "babel-jest",
-      "\\.css$": "<rootDir>/cssTransform.js"
```

Comment:
?

Reply:
I took a while for thinking of how can I design this code fragment.
It argued in our team, but I reckon cssTransform.js has no test code before and it passes the test through the human-artifact condition to filter out the report instead.
Once the condition removed, the phenomenon comes out.

Reply:
This test code is recovered in the latest push, and we show the original code coverage in snapshot instead.

Reply:
I took a while for thinking of how can I design this code fragment.
It argued in our team, but I reckon cssTransform.js has no test code before and it passes the test through the human-artifact condition to filter out the report instead.
Once the condition removed, the phenomenon comes out.

Reply:
This test code is recovered in the latest push, and we show the original code coverage in snapshot instead.